src/components/Card/Card.jsx

```diff
@@ -1,25 +1,39 @@
+import { Component } from 'react';
+import { CartContext } from 'contexts/CartContext.jsx';
 import Button from 'components/Button/Button.jsx';
 import styles from './Card.module.css';
 
-const Card = ({ name, price, description, image }) => {
-    return (
-        <div className={styles.card}>
-            <div className={styles.imageContainer}>
-                <img src={image} alt={name} />
-            </div>
-            <div className={styles.cardContent}>
-                <div className={styles.cardHeader}>
-                    <div className={styles.cardName}>{name}</div>
-                    <div className={styles.cardPrice}>{price}</div>
+class Card extends Component {
+    static contextType = CartContext;
+
+    handleAddToCart = () => {
+        const { incrementCartCount } = this.context;
+        incrementCartCount();
+    };
+
+    render() {
+        const { item } = this.props;
+        const { meal, price, instructions, img } = item;
```

Comment:
You could get an access to properties in one line like const {item: {meal, ...rest}} = this.props.